README.md

```diff
@@ -76,6 +76,34 @@ In the future [WGSL](https://gpuweb.github.io/gpuweb/wgsl.html) will be the shad
 
 For now, the shaders can be compiled to SPIR-V by running `make`, which requires you to have `glslang`s `glslangValidator` binary.
 
+## Logging
+
+`wgpu-core` uses `tracing` for logging and `wgpu-rs` uses `log` for logging. 
```

Comment:
maybe a comma is missing before "and"?

Reply:
I was always taught that `and` doesn't get a comma when used with only two elements, but english comma rules are nothing if not confusing

Reply:
I'm not nearly a native speaker, so please free to ignore this suggestion :)